### DISCUSSION TITLE
Adding large images in HTML

### DISCUSSION THREAD

**Discussion by anjanesh** [mpdf/mpdf#1811] [Q&A]
Lets say I have an HTML with 2 images needed in IMG src.

1. These images are 10 MB each.
2. These images are not on the global network - only localhost

How do I send these images to mPDF ?

Is it possible to do a combination of :

`$mpdf->Image();` and `$mpdf->WriteHTML($html);` ?

**Comment by aminhaiqal** [comment]
Yes, as far as I know it possible to do a combination of `$mpdf->Image();` and `$mpdf->WriteHTML($html);`

**Reply by anjanesh** [reply]
Does it better sense to convert each image to base64 ?

**Reply by aminhaiqal** [reply]
Depends on your purpose. 
You can convert image to base64 if you want to include them in PDF. For sure it will fasten the process time.
However, its not the best practice if you want to show them in HTML cause it will increase the html size and page loading time.

**Reply by anjanesh** [reply]
There are like 50 images with each image being 10MB and all I need is PDF for now - so the HTML content sent to mPDF would be huge I guess.
